src/main/java/com/nexenio/CppModernJson/CppModernJsonCodegen.java

```diff
@@ -230,7 +246,13 @@ public CodegenModel fromModel(String name, Schema model) {
     Set<String> oldImports = codegenModel.imports;
     codegenModel.imports = new HashSet<>();
     for (String imp : oldImports) {
-      String newImp = toModelImport(imp);
+      String newImp = "";
+      if(imp.startsWith("std::shared_ptr")) {
```

Comment:
Might be a bit naïve, but: Is there a better way to figure this out than fishing for `"std::shared_ptr"`? Seems a bit fragile to me?!

Reply:
We don't have type information here. Only strings. Do you have something in mind?